mllib/src/main/scala/org/apache/spark/mllib/regression/Lasso.scala

```diff
@@ -85,6 +85,7 @@ object LassoModel extends Loader[LassoModel] {
  * See also the documentation for the precise formulation.
  */
 @Since("0.8.0")
+@deprecated("Use ml.regression.LinearRegression with elasticNetParam = 1.0", "2.0.0")
```

Comment:
We should also notice ```with regParam = 0.01``` because the default value of ```regParam``` is 0.0 which will make ```ml.regression.LinearRegression``` training without regularization.